tools/cockpit.spec

```diff
@@ -81,8 +81,8 @@ Source0:        https://github.com/cockpit-project/cockpit/releases/download/%{v
 %define build_optional 1
 %endif
 
-# Ship custom SELinux policy only in Fedora and RHEL-9 onward
-%if 0%{?rhel} >= 9 || 0%{?fedora}
+# Ship custom SELinux policy on Fedora and RHEL/CentOS
+%if 0%{?rhel} || 0%{?fedora}
```

Comment:
...so everywhere, then?

Can we drop the `%if` entirely?  Or are you thinking about other distros?

Reply:
The main distro which we support with our .spec is OpenSUSE. I have zero ideas how SELinux is being managed there, and we don't have it in CI so I don't dare to do this big change to it without at least consulting @nussel

Reply:
Thanks for the consideration! You really don't have to do that though. Just don't worry. If it breaks I'll fix it. As a matter of fact I've patched that %if condition to also be true on openSUSE. I couldn't send the patch yet as IIRC futher down the spec file expects setroubleshoot which is still stuck in the review queue.

Reply:
Thanks @lnussel, so much the better! Then I'll just drop the whole `%{with_selinux}` then, which should make the spec simpler.

Reply:
Eh sorry, no -- we still have the cockpit-appstream hack for RHEL 8, so `%{with_selinux}` will have to stay for a while, I'm afraid. But at least the marked condition above can go.

Reply:
Dropped.